### PR TITLE
Task black formatter +  flake8 checker added

### PR DESCRIPTION
**Pull request by dimacodota**

### CODE REVIEW

**Review by boaz-codota — CHANGES_REQUESTED** [review]

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+repos:
+    -   repo: https://github.com/ambv/black
```
[the remark this answers is not in the document]
Comment (dimacodota) [reply .pre-commit-config.yaml:2]: on it

[the remark this answers is not in the document]
Comment (dimacodota) [reply .pre-commit-config.yaml:2]: done

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[tool.black]
```
[the remark this answers is not in the document]
Comment (dimacodota) [reply pyproject.toml:1]: no it is managed by pre-commit
I've added the installation instructions, it just `pip install -r requirements-dev.txt`

File: .flake8 [hunk 3]
```diff
@@ -0,0 +1,5 @@
+[flake8]
+max-line-length = 88
```
[the remark this answers is not in the document]
Comment (dimacodota) [reply .flake8:2]: it is default

**Review by boaz-codota — APPROVED** [review]

**Pull request merged**